kahuna/public/js/leases/leases.html

```diff
@@ -123,8 +123,26 @@
     {{ctrl.activeLeases(ctrl.leases)}} current leases + {{ctrl.inactiveLeases(ctrl.leases)}} inactive leases
   </div>
 
-  <ul ng-if="ctrl.totalImages === 1">
-    <li ng-repeat="lease in ctrl.leases.leases"
+  <ul ng-if="ctrl.totalImages === 1" ng-switch="ctrl.leasesUpdatedByTemplate">
+    <li ng-switch-when="true"
+        ng-repeat="lease in ctrl.updatedLeases"
+        class="lease__item"
+        ng-class="{'lease__item-preview' : lease.fromTemplate}">
+        <div class="lease__current" ng-class="ctrl.leaseStatus(lease).current" ng-if="ctrl.leaseStatus(lease).current"></div>
+        <div class="lease__access" ng-class="ctrl.leaseStatus(lease).access"></div>
+        <div class="lease__wrapper"  ng-class="ctrl.leaseStatus(lease).current">
```

Comment:
```suggestion
        <div class="lease__wrapper" ng-class="ctrl.leaseClass(lease)">
```

an update to keep this in sync with the current leases component